src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerGenotypingEngine.java

```diff
@@ -218,6 +210,16 @@ public CalledHaplotypes assignGenotypeLikelihoods(final List<Haplotype> haplotyp
         return new CalledHaplotypes(phasedCalls, calledHaplotypes);
     }
 
+    protected VariantContext makeAnnotatedCall(byte[] ref, SimpleInterval refLoc, FeatureContext tracker, SAMFileHeader header, VariantContext mergedVC, ReadLikelihoods<Allele> readAlleleLikelihoods, VariantContext call) {
+        final SimpleInterval locus = new SimpleInterval(mergedVC.getContig(), mergedVC.getStart(), mergedVC.getEnd());
+        final SimpleInterval refLocInterval= new SimpleInterval(refLoc);
+        final ReferenceDataSource refData = new ReferenceMemorySource(new ReferenceBases(ref, refLocInterval), header.getSequenceDictionary());
+        final ReferenceContext referenceContext = new ReferenceContext(refData, locus, refLocInterval);
+        final VariantContext untrimmedResult =  annotationEngine.annotateContext(call, tracker, referenceContext, readAlleleLikelihoods.toPerReadAlleleLikelihoodMap(), a -> true);
+        return call.getAlleles().size() == mergedVC.getAlleles().size() ? untrimmedResult
+                : GATKVariantContextUtils.reverseTrimAlleles(untrimmedResult);
```

Comment:
I don't know.  This is copied from existing behavior.  I promised David Roazen not to affect HaplotypeCaller in any way.